Added alpine support

alrayyes opened the pull request.

alrayyes committed:
Added alpine support

alrayyes commented:
Hope this works as I couldn't find decent documentation on how to test local Wercker steps.

ArjenSchwarz commented:
Maybe it works like this. You try to install pygments twice instead of making it mercurial one time

alrayyes committed:
install go/git/mercurial instead python-pygments again in Alpine

Should address issue https://github.com/ArjenSchwarz/wercker-step-hugo-build/pull/24#issuecomment-249182033

alrayyes commented:
Should be fixed in 4608348af8cf82cb872ed404da4b3d52903444b7
